content/rs/administering/database-operations/slave-ha.md

```diff
@@ -61,14 +61,13 @@ make sure that its slave shards are created first. To assign priority to a datab
 
 1. Slave shards of CRDBs - because the sync between the various replicas is critical, 
 and should be performed using slave shards.
-1. Cache databases - because such databases are unavailable if it is not moved to an available node.
```

Comment:
@Paz-Yanover Did you intentionally remove cache databases?

Reply:
Yes